app-emulation/lxd/lxd-0.18.ebuild

```diff
@@ -0,0 +1,172 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+DESCRIPTION="Fast, dense and secure container management"
+HOMEPAGE="https://linuxcontainers.org/lxd/introduction/"
+EGO_PN_PARENT="github.com/lxc"
+EGO_PN="${EGO_PN_PARENT}/lxd"
+SRC_URI="http://961db08fe45d5f5dd062-b8a7a040508aea6d369676e49b80719d.r29.cf2.rackcdn.com/${P}.tar.bz2"
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+
+PLOCALES="de fr ja"
+IUSE="btrfs +criu +daemon lvm nls test"
+
+# IUSE and PLOCALES must be defined before l10n inherited
+inherit bash-completion-r1 eutils golang-build l10n systemd user vcs-snapshot
+
+DEPEND="
+	dev-go/go-crypto
+	>=dev-lang/go-1.4.2:=
+	dev-libs/protobuf
+	dev-vcs/git
+	nls? ( sys-devel/gettext )
+	test? (
+		app-misc/jq
+		dev-db/sqlite
+		net-misc/curl
+		sys-devel/gettext
+	)
+"
+
+RDEPEND="
+	daemon? (
+		app-admin/cgmanager
+		app-arch/xz-utils
+		app-emulation/lxc[cgmanager]
+		net-analyzer/openbsd-netcat
+		net-misc/bridge-utils
+		net-misc/rsync[xattr]
+		sys-apps/iproute2
+		virtual/acl
+		btrfs? (
+			sys-fs/btrfs-progs
+		)
+		criu? (
+			sys-process/criu
+		)
+		lvm? (
+			sys-fs/lvm2
+		)
+	)
+"
+
+# KNOWN ISSUES:
+# - Translations may not work.  I've been unsuccessful in forcing
+#   localized output.  Anyway, upstream (Canonical) doesn't install the
+#   message files.
+
+# TODO:
+# - since 0.15 gccgo is a supported compiler ('make gccgo').  It would
+#   be preferable for that support to go into the golang-build eclass not
+#   this package directly.
+# - Add apparmor USE.  There are some exec calls to apparmor_parser
+# - Test build with Go 1.4 & 1.5
+# - Add seccomp USE - upstream release notes for 0.18 mention it, a global
+#   USE flag exists
+# - check daemon flags, need --force-local in conf.d?
+# - where is --force-local (0.18)? not documented in --help
+# - integrate "lxd shutdown" into initscript as custom action (default "stop"
+#   action should _not_ stop containers amirite?)
+#   "Perform a clean shutdown of LXD and all running containers"
+
+src_prepare() {
+	cd "${S}/src/${EGO_PN}"
+
+	epatch "${FILESDIR}/${P}-dont-go-get.patch"
+
+	if use daemon; then
```

Comment:
Does patching need to be behind a USE flag condition?  Seems like we could unconditionally apply the patch.  What do you think?